packages/config-manager/src/genesis.ts

```diff
@@ -0,0 +1,70 @@
+import { Block, utils } from "@ckb-lumos/base";
+import { ScriptConfig } from "./types";
+
+// https://github.com/nervosnetwork/ckb-sdk-rust/blob/94ce4379454cdaf046f64b346e18e73e029f0ae6/src/constants.rs#L19C1-L24C62
```

Comment:
maybe we don't need to fully import type here like `usize` in rust.

```
type TransactionIndex = number;
type InputIndex = number;
type SystemScriptLocationInfo = [TransactionIndex, InputIndex]
export const SIGHASH_OUTPUT_LOC: SystemScriptLocationInfo = [0, 1];
```
seems more readable

Reply:
improved the readability